### ISSUE TITLE
Meaning of empty contour?

### ISSUE THREAD

**Issue by madig** [unified-font-object/ufo-spec#150]
According to https://unifiedfontobject.org/versions/ufo3/glyphs/glif/#child-elements-2, a `<contour>` may contain nothing (e.g. `<contour/>`). ufoLib dutifully parses this and ufoLib2 makes 

```
In [5]: glyph.contours
Out[5]: [Contour(points=[])]
```

out of it. I wonder if an empty contour has any meaning though? If not, should it be pruned/skipped?

**Comment by alerque** [comment]
**If** it has no meaning (an "if" that somebody more knowledgeable than I will need to speak to), then pruning it should be part of the recommended normalization process.

**Comment by anthrotype** [comment]
I think the parent `<outline/>` may be empty (i.e. neither contours nor components), but an empty `<contour/>` simply doesn't make any sense and should be treated as if it was not there.

**Comment by madig** [comment]
I can imagine there being some sort of meaing if one assigns an identifier to an empty outline to maybe programmatically do something or assign some meaning to it via public object lib key but eh?

**Comment by anthrotype** [comment]
what does ufoNormalizer do in this case?

**Comment by anthrotype** [comment]
(sorry, I clicked the wrong button)

**Comment by madig** [comment]
It removes all of 

```
		<contour identifier="aaaa"/>

		<contour/>

		<contour>
		</contour>

		<contour identifier="vvv">
		</contour>
```

**Comment by typesupply** [comment]
I may have had a reason to allow 0 child elements in the spec, but I don't remember why. If the "remove empty contour elements" part of ufoNormalizer came from me, I'd defer to that because I did a lot of thinking about everything then. Can someone see if I was the one who added the [`if not points` condition](https://github.com/unified-font-object/ufoNormalizer/blob/d22e855b395850893324e8aac8249a916375cee6/src/ufonormalizer.py#L983)? I took a quick browse through the history manually but I couldn't find when that came in. I don't know how to do this programatically.
